src/assets/styles/global.scss

```diff
@@ -23,9 +23,6 @@
   overflow: hidden;
 }
 
-[calcite-hydrated] {
```

Comment:
is this added some other way, now?

Reply:
nvm I see that stencil is adding this style to the tag it's injecting into the head. 

Outside the scope of this pr, but if we could figure it out, a better way to handle this would be to do an all-CSS solution in our global sheet like:


```
calcite-tabs:not([calcite-hydrated]),
calcite-button:not([calcite-hydrated]),
... { 
  visibility: hidden;
}
```

That way this rule wouldn't change the fundamental display property of elements and you could still hide elements from their css sheet if you needed to. Also, right now elements will be available until JS finishes parsing and creates that tag in our head, so for slower environments, you will see a flash of unstyled, raw html of a components slot content.

Reply:
ah cool. so this PR can proceed

Reply:
👍 merging